analysis_plan.md

```diff
@@ -0,0 +1,9 @@
+Project OUTLIERS
+
+(1) Adapt DVARS from exercices in findoutlie/metrics.py
+(2) Fill in findoutlie/detectors.py for IQR
+(3) Fill in findoutlie/outfind.py 
+(4) Test on one subject
+(5) Test on the 9 other subjects 
+(6) Add other metrics from MRIqc : framewise displacement, tSNR
```

Comment:
Point number 6 has two issues:
- If you are not going to run MRIQC, but rather implement these metrics then it's better to remove MRIQC from the plan and think about what will be necessary to calculate these.
- If you are going to run MRIQC, then why using just these two?

```suggestion
(6) Add other metrics:
    (6.1) framewise displacement -> requires head motion estimation
    (6.2) tSNR -> requires (at the very least) brain extraction, but it is better if you can calculate it in ROIs more targetted.
```